src/plugins/options/options.ts

```diff
@@ -24,39 +24,20 @@ export class UIPOptions extends UIPPlugin {
 
   protected bindEvents() {
     this.addEventListener('change', this._onOptionChange);
-    this.root?.addEventListener('uip:configchange', this._onRootConfigChange);
   }
 
   protected unbindEvents() {
     this.removeEventListener('change', this._onOptionChange);
-    this.root?.removeEventListener('uip:configchange', this._onRootConfigChange);
   }
 
   protected render() {
     this.innerHTML = '';
-    this.renderMode();
     this.renderTheme();
+    this.renderSettingsControl();
+    this.renderEditorControl();
```

Comment:
As far as i can see, settings and editor options render any time, even if one of them is missing in playground ( as far as i also remember settings aren't planning to be present in esl demosite for the first time). We should create a way to  let options know, whether we want editor/settings option control, maybe through attribute

Reply:
I believe we will implement this things in next option update.